types/glob-watcher/glob-watcher-tests.ts

```diff
@@ -0,0 +1,60 @@
+import * as fs from 'fs';
+import * as globWatcher from 'glob-watcher';
```

Comment:
namespaces are not collable. Instead use
```suggestion
import globWatcher = require('glob-watcher');
```

Reply:
It's callable because the namespace shares the same name with the two function definition exports at the top of index.d.ts, line 16 and line 24.
The package being modeled is a single function export.

Reply:
Yeah I noticed that but it's a bad practice and comes with a couple of problems. You can read about it on the definitively typed [FAQ here](https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports). Also there's a very complete post explaining a little bit more about this [here](https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this).